README.md

```diff
@@ -5,7 +5,15 @@
 This is the source for [**Santiment Technical Documentation site**](https://academy.santiment.net)
 
 ## Running locally
-
+Yarn installation can be done this way on Ubuntu:  
```

Comment:
@Snedashkovsky I am sure we don't need to give OS related information about how to install yarn. And any information about installing yarn cause it's not our software, user can find related info in yarn documentation.

Reply:
@Partysun I spent over 20 minutes on installing this application, hopefully my description will help save time for someone else.

Reply:
@Snedashkovsky I found all you lines in yarn official site (including special command `sudo apt remove cmdtest`). Link to this site is enough for users who want to install it.